src/content/docs/logs/forward-logs/forward-your-logs-using-infrastructure-agent.mdx

```diff
@@ -71,87 +45,19 @@ Here's an example of logs for your host's UI. You can see logs in context of eve
 />
 
 <figcaption>
-  Here's an example of a host's logs in context of an event.
+From your host's UI, your logs are placed in context of events for the selected time period. You can drill down into detailed data for any of the highlighted attributes.
 </figcaption>
 
-## Enable logging for your on-host integrations [#on-host]
 
-With the infrastructure agent installed, you can enable automatic log parsing and forwarding for our most popular on-host integrations with one step. To enable this feature, rename the `on-host-log.yml.example` file to `on-host-log.yml`. Once done, your integration's logs are automatically parsed and sent to New Relic.
+## Install the infrastructure agent [#infra]
 
-This option is available for our [supported Linux platforms](#requirements).
-
-To enable the on-host integration log forwarding feature:
+Starting with version 1.11.4, the infrastructure agent forwards logs to New Relic. You can install the infrastructure agent with our guided install, or with a manual install. Our guided install is a lightweight route that installs our infrastructure agent, an APM agent, and more. This is a good choice for smaller organizations or any organization that wants to test out New Relic. 
```

Comment:
Do we need to put the version where this feature started here? Just asking for curiosity's sake

Reply:
I don't think you need a comma in the second sentence. Something like "guided or manual install" might work better and eliminates minor repetition.